packages/frontpage/package.json

```diff
@@ -32,9 +33,9 @@
     "date-fns": "^3.6.0",
     "drizzle-orm": "^0.31.2",
     "jose": "^5.4.0",
-    "next": "15.0.0-rc.0",
-    "next-auth": "5.0.0-beta.19",
+    "next": "15.0.0-canary.117",
```

Comment:
I probably dont need to upgrade next.js, I only upgraded because the newer canaries have fixes for vscode debugging.

When I'm comfortable everything mostly works I'll look at keeping the current rc in this PR.